.github/workflows/ci-release.yaml

```diff
@@ -0,0 +1,48 @@
+name: CI Release Action
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+      - "v*.*.*-rc.*"
```

Comment:
if tags is `v*.*.*-rc.*` then do we want it to be a draft release? Because drafts are not visible for let's say GitOps Controllers

Reply:
I would still love to do draft release first, because there we have the chance to fix issues in the background or link/add/remove things.